pkg/entities/webhook.go

```diff
@@ -443,6 +467,14 @@ func (e *Entity) WebhookUpvoteStreak(userID, source string) error {
 			e.log.Errorf(err, "[e.WebhookUpvoteStreak] fail to create new streak")
 			return fmt.Errorf("failed to create new user upvote streak: %v", err)
 		}
+		// handle quest logs
+		log := &model.QuestUserLog{
+			UserID: userID,
+			Action: model.QuestAction(model.VOTE),
+		}
+		if err := e.UpdateUserQuestProgress(log); err != nil {
+			e.log.Fields(logger.Fields{"log": log}).Error(err, "[entity.WebhookUpvoteStreak] entity.UpdateUserQuestProgress() failed")
+		}
```

Comment:
should we return here

Reply:
return err here may prevent below functions from execution
this quest handler should not affect original function flow imo

Reply:
same as above